assets/xml/objects/gameplay_keep.xml

```diff
@@ -766,59 +766,59 @@
-        <Limb Name="gameplay_keep_Standardlimb_011484" Type="Standard" Offset="0x11484" />
-        <Limb Name="gameplay_keep_Standardlimb_011490" Type="Standard" Offset="0x11490" />
-        <Limb Name="gameplay_keep_Standardlimb_01149C" Type="Standard" Offset="0x1149C" />
-        <Limb Name="gameplay_keep_Standardlimb_0114A8" Type="Standard" Offset="0x114A8" />
-        <Limb Name="gameplay_keep_Standardlimb_0114B4" Type="Standard" Offset="0x114B4" />
-        <Limb Name="gameplay_keep_Standardlimb_0114C0" Type="Standard" Offset="0x114C0" />
-        <Limb Name="gameplay_keep_Standardlimb_0114CC" Type="Standard" Offset="0x114CC" />
-        <Limb Name="gameplay_keep_Standardlimb_0114D8" Type="Standard" Offset="0x114D8" />
-        <Limb Name="gameplay_keep_Standardlimb_0114E4" Type="Standard" Offset="0x114E4" />
-        <Skeleton Name="gameplay_keep_Skel_011518" Type="Normal" LimbType="Standard" Offset="0x11518" />
-        <Animation Name="gameplay_keep_Anim_0117A8" Offset="0x117A8" />
-        <Collision Name="gameplay_keep_Colheader_0118D8" Offset="0x118D8" />
-        <DList Name="gameplay_keep_DL_011BD0" Offset="0x11BD0" />
-        <DList Name="gameplay_keep_DL_0121B0" Offset="0x121B0" />
-        <DList Name="gameplay_keep_DL_012260" Offset="0x12260" />
-        <DList Name="gameplay_keep_DL_012308" Offset="0x12308" />
-        <DList Name="gameplay_keep_DL_012390" Offset="0x12390" />
-        <DList Name="gameplay_keep_DL_012418" Offset="0x12418" />
-        <DList Name="gameplay_keep_DL_0124A0" Offset="0x124A0" />
-        <DList Name="gameplay_keep_DL_012528" Offset="0x12528" />
-        <DList Name="gameplay_keep_DL_0125B0" Offset="0x125B0" />
-        <DList Name="gameplay_keep_DL_012638" Offset="0x12638" />
-        <DList Name="gameplay_keep_DL_0126C0" Offset="0x126C0" />
-        <Limb Name="gameplay_keep_Standardlimb_012748" Type="Standard" Offset="0x12748" />
-        <Limb Name="gameplay_keep_Standardlimb_012754" Type="Standard" Offset="0x12754" />
-        <Limb Name="gameplay_keep_Standardlimb_012760" Type="Standard" Offset="0x12760" />
-        <Limb Name="gameplay_keep_Standardlimb_01276C" Type="Standard" Offset="0x1276C" />
-        <Limb Name="gameplay_keep_Standardlimb_012778" Type="Standard" Offset="0x12778" />
-        <Limb Name="gameplay_keep_Standardlimb_012784" Type="Standard" Offset="0x12784" />
-        <Limb Name="gameplay_keep_Standardlimb_012790" Type="Standard" Offset="0x12790" />
-        <Limb Name="gameplay_keep_Standardlimb_01279C" Type="Standard" Offset="0x1279C" />
-        <Limb Name="gameplay_keep_Standardlimb_0127A8" Type="Standard" Offset="0x127A8" />
-        <Limb Name="gameplay_keep_Standardlimb_0127B4" Type="Standard" Offset="0x127B4" />
-        <Skeleton Name="gameplay_keep_Skel_0127E8" Type="Flex" LimbType="Standard" Offset="0x127E8" />
+        <Collision Name="gPinkDekuFlowerCol" Offset="0xE710" />
```

Comment:
Hmm, I wonder if we should have newlines where the things aren't related like you did at the bottom of this block. This xml is already huge.

Reply:
Huh, I don't see any newlines on my end. Do you want me to insert a newline (and maybe a comment) at the start and end of the Deku Flower section?

Reply:
Oh, it's GitHub's pagewidth playing with the xml tags. Yes, I think that would be a good idea, and head the section with a comment.